<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPointsAttributesManager</class>
 <widget class="QDialog" name="GPointsAttributesManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configurer les couleurs des points</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
     <item>
      <widget class="QWidget" name="widgetEditGradient" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="GColorGradientView" name="colorGradientView" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QtColorPicker" name="pushButtonGradientColorPicker">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="GGradientManager" name="gradientManagerView" native="true">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBoxGradientArrowValue">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAddColor">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Ajouter une couleur</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDeleteColor">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Supprimer la couleur</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSave">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Enregistrer</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widgetEditNormals" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Taille des normales</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxNormalsLength">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Couleur des normales</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QtColorPicker" name="pushButtonNormalsColorPicker">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QPushButton" name="pushButtonApplyEditNormals">
          <property name="text">
           <string>Appliquer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBoxShowNormals">
         <property name="text">
          <string>Utiliser les normales d√©finies par les attributs</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GGradientManager</class>
   <extends>QWidget</extends>
   <header>view/Tools/gradient/ggradientmanager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GColorGradientView</class>
   <extends>QWidget</extends>
   <header>view/DocumentView/GraphicsViews/ColorGradient/gcolorgradientview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtColorPicker</class>
   <extends>QPushButton</extends>
   <header>qtcolorpicker/qtcolorpicker.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
